core-profile-tck/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
fyi for other reviewers, https://blogs.eclipse.org/post/wayne-beaton/eclipse-project-licenses states that Eclipse projects already are using `Apache License Version 2.0`